scripts/install/nginx.sh

```diff
@@ -57,7 +57,7 @@ else
   geoip=php-geoip
 fi
 
-if [[ $codename =~ ("bionic"|"buster") ]]; then
+if [[ $codename =~ ("bionic"|"buster"|"focal") ]]; then
```

Comment:
I think you are actually correct here. If mcrypt is gone then the list of distros which support it will soon diminish to none once Jessie and Xenial are EOL